test/fileio.jl

```diff
@@ -50,6 +50,29 @@ let fn = tempname() * ".h5"
   dat = load(fn; dict=OrderedDict())  # `track_order` is inferred from `OrderedDict`
 
   @test all(keys(dat) .== ["b", "a", "G/z", "G/f"])
+
+  # issue #939
+  h5open(fn, "r"; track_order=true) do io
+    @test HDF5.get_create_properties(io).track_order
```

Comment:
```suggestion
    @test HDF5.get_context_property(:file_create).track_order
```

Reply:
This is clearer to me that the property is coming from the context rather than the object instance `io`.